docs/guide/data_migration/add_data_migration_matcher.md

```diff
@@ -0,0 +1,47 @@
+# Add data migration matcher
+
+[Matchers in data migrations](exporting_and_importing_data.md#match-property) enable you to select which data from the Repository to export.
+
+In addition to the built-in matchers, you can create custom matchers for content.
+
+The following example create a matcher for Section identifiers.
+
+## Create normalizer
+
+To do this, first add a normalizer which handles the conversion between objects and the YAML format used for data migration.
+
+!!! tip "Normalizers"
+
+    To learn more about normalizers, refer to [Symfony documentation]([[= symfony_doc =]]/components/serializer.html).
+
+Create the normalizer in `src/Migrations/Matcher/SectionIdentifierNormalizer.php`:
+
+``` php
+[[= include_file('code_samples/data_migration/src/Migrations/Matcher/SectionIdentifierNormalizer.php') =]]
+```
+
+Register the normalizer as a service:
+
+``` yaml
+[[= include_file('code_samples/data_migration/config/custom_services.yaml', 10, 13) =]]
+```
+
+!!! note "Normalizer order"
+
+    User-defined normalizers are always executed before the built-in ones.
+    However, you can additionally set the priority of your normalizers.
```

Comment:
Normalization service priorities can be checked using `php bin/console debug:container --tag ibexa.migrations.serializer.normalizer`